test/test_genome.py

```diff
@@ -548,3 +548,71 @@ def test_mutate_does_not_reinitialize_parameters(genome_params, rng, mutation_ra
     assert genome._parameter_names_to_values["<p2>"] == pytest.approx(
         parameter_names_to_values_before["<p2>"]
     )
+
+
+def test_genome_reordering_empirically(rng):
```

Comment:
just to be safe, can you add tests that check that an error is raised when one is trying to apply reorder to a genome where `n_rows != 1` and/or `n_levels_back != n_columns`? you can do that with the `pytest.raises(...)` context manager

Reply:
(ping)

Reply:
sorry, pushed the wrong commit yesterday, it will be in the reworked version